Sources/Sentry/SentrySerialization.m

```diff
@@ -339,6 +339,24 @@ + (SentryAppState *_Nullable)appStateWithData:(NSData *)data
     return [[SentryAppState alloc] initWithJSONObject:appSateDictionary];
 }
 
++ (NSDictionary *)eventEnvelopeItemJson:(NSData *)eventEnvelopeItemData
```

Comment:
`m`: This method is basically only a wrapper around `NSJSONSerialization JSONObjectWithData`. I think we should change the name to be more generic, as it doesn't really convert NSData to an envelopeItemJSON.